specification.md

```diff
@@ -3126,8 +3126,8 @@ Delay state waits for a certain amount of time before transitioning to a next st
 | name | State name | string | yes |
 | type | State type | string | yes |
 | [branches](#parallel-state-branch) | List of branches for this parallel state| array | yes |
-| completionType | Option types on how to complete branch execution. Default is "and" | enum | no |
-| n | Used when branchCompletionType is set to `n_of_m` to specify the `n` value. | string or number | no |
+| completionType | Option types on how to complete branch execution. Default is "allOf" | enum | no |
+| numCompleted | Used when branchCompletionType is set to `atLeast` to specify the number of branches that must complete. | string or number | no |
```

Comment:
the least number of branches that must complete in order for the state to transition / end